client-app/src/pages/account/store/modules/orders-list/getters.ts

```diff
@@ -16,5 +17,12 @@ export const getters: GetterTree<OrdersListState, RootState> = {
   },
   orderIsLoaded: (state: OrdersListState): boolean => {
     return state.selectedOrderIsLoaded;
+  },
+  getDatepickerLocales: (state: OrdersListState): LocaleMessages | {} => {
```

Comment:
It is the better will be set localization for date picker directly in the component controller, do not overload the state with data not related to business logic and interactions with API.